notifications_utils/logging.py

```diff
@@ -89,7 +89,12 @@ def after_request(response):
 
     handlers = get_handlers(app, extra_filters=extra_filters)
     loglevel = logging.getLevelName(app.config["NOTIFY_LOG_LEVEL"])
-    loggers = [app.logger, logging.getLogger("utils")]
+    loggers = [
+        app.logger,
+        logging.getLogger("utils"),
+        logging.getLogger("celery.worker"),
+        logging.getLogger("celery.redirected"),  # stdout/stderr
```

Comment:
How did you find/pick these 2 logger names?

I can see `celery.redirected` being referenced in the celery code (https://github.com/celery/celery/blob/v5.2.7/celery/app/log.py#L80)

I can also see `celery` and `celery.task` being referenced in that same file for logger names but can't see any `celery.worker` anywhere?

Reply:
https://github.com/celery/celery/blob/32a285dd956756322e8a9c9310731e8419a3660b/celery/utils/log.py#L106

Reply:
FWIW longer-term I think we should consider consuming the `setup_loggers` signal, preventing https://github.com/celery/celery/blob/32a285dd956756322e8a9c9310731e8419a3660b/celery/app/log.py#L102 from running and allowing us to set up the loggers ourselves without interference. But that would need more fiddling & testing.